Source/Blazorise/Components/DatePicker/DatePicker.razor.cs

```diff
@@ -502,9 +506,14 @@ protected override IReadOnlyList<TValue> InternalValue
     [Inject] protected ITextLocalizer<DatePicker<TValue>> Localizer { get; set; }
 
     /// <summary>
-    /// Converts the supplied date format into the internal date format.
+    /// Converts the supplied date format into the internal date format used by the <see cref="DisplayFormat"/> mask.
     /// </summary>
-    [Inject] protected IDateTimeFormatConverter DateTimeFormatConverter { get; set; }
+    [Inject] protected IDateTimeDisplayFormatConverter DisplayFormatConverter { get; set; }
```

Comment:
Aren't these converters just because of how the 3rd party library for the pickers handle the formats, since they are not the same as the dotnet formats?
Should the naming not reflect that? Otherwise it looks like a contract that can be used elsewhere.

Reply:
You mean like IFlatPickrDateTimeDisplayFormatConverter ?

Reply:
For example. It's not like the purpose is going to be anything else. That's way clearer to me.